### DISCUSSION TITLE
Is it possible to select an attribute of a parent element as a description in Picviewer?

### DISCUSSION THREAD

**Discussion by Omaeda** [hoothin/UserScripts#800] [Q&A] [answered]
this is the rule I am currently using:
```
{
      name: "donmai",
      url: /donmai\.us/,
      src: /.us\/(?:.+?)\/(.*)\/([0-9a-fA-F]+).jpg/,
      r: /.us\/(?:.+?)\/(.*)\/([0-9a-fA-F]+).jpg/,
      s: [".us/sample/$1/sample-$2.jpg", ".us/original/$1/$2.png",".us/original/$1/$2.jpg"],
      description: "./../../../../@data-tags"
}
```

And this is the error I get

![image](https://github.com/user-attachments/assets/c47d475e-ccf5-42c7-aae6-a11d07415842)

**Comment by hoothin** [comment, marked as answer]
The `description` should point to an element; if it does not point to the `title`, it needs to use an array. Like `description: ["./../../../..", "data-tags"]`
